test/classes/override/override-mix.good

```diff
@@ -3,8 +3,4 @@ override-mix.chpl:11: error: Child.g override keyword present but no superclass
 override-mix.chpl:11: note: type for argument 'arg' does not match
 override-mix.chpl:3: note: base method uses 'int(32)'
 override-mix.chpl:11: note: overriding method uses 'int(16)'
-override-mix.chpl:12: error: Child.g override keyword present but no superclass method matches signature to override
-override-mix.chpl:12: note: type for argument 'arg' does not match
-override-mix.chpl:3: note: base method uses 'int(32)'
-override-mix.chpl:12: note: overriding method uses 'int(64)'
```

Comment:
Is the change here because we only resolve `override` mismatches when a call to that override exists?

Reply:
yes